src/main/java/org/jitsi/videobridge/octo/OctoRelayService.java

```diff
@@ -83,15 +80,15 @@ public OctoRelay getRelay()
     @Override
     public void start(BundleContext bundleContext)
     {
-        cfg
-            = ServiceUtils.getService(
+        ConfigurationService cfg
+            = ServiceUtils2.getService(
                     bundleContext, ConfigurationService.class);
 
         String address = cfg.getString(ADDRESS_PNAME, null);
         String publicAddress = cfg.getString(PUBLIC_ADDRESS_PNAME, address);
         int port = cfg.getInt(PORT_PNAME, -1);
 
-        if (address != null && NetworkUtils.isValidPortNumber(port))
+        if (address != null && (1024 <= port && port <= 0xffff))
```

Comment:
no more access to `NetworkUtils`?

Reply:
No, and that was the only place where `jitsi-netaddr` was used.